app/models/user.rb

```diff
@@ -9,4 +9,30 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+
+  has_many :friendships
+  has_many :inverse_friendships, class_name: 'Friendship', foreign_key: 'friend_id'
+
+  def friends
+    friends_list = friendships.map { |friendship| friendship.friend if friendship.status }
+    friends_list.compact
+  end
```

Comment:
To turn this method into an association you will need:
- `has_many :confirmed_friendships`. Where you select only the friendships that have the `status: true`.  HINT: you need to declare the `class` to set this association.
-  `has_many :friends`. HINT: this is a through association that uses the previous one.